presto-main/src/main/java/io/prestosql/operator/WindowOperator.java

```diff
@@ -285,8 +293,7 @@ public WindowOperator(
                     orderChannels,
                     ordering,
                     spillerFactory,
-                    // TODO use compiled comparator
-                    new SimplePageWithPositionComparator(sourceTypes, unGroupedOrderChannels, unGroupedOrdering)));
+                    orderingCompiler.compilePageWithPositionComparator(sourceTypes, unGroupedOrderChannels, unGroupedOrdering)));
```

Comment:
Update err msg in `io.prestosql.sql.gen.OrderingCompiler#internalCompilePageWithPositionComparator`, as it currently says "Error compiling **merge sort** comparator"